framework/wazuh/core/cluster/cluster.py

```diff
@@ -406,28 +405,32 @@ def split_on_condition(seq, condition):
     # Missing files will be the ones that are present in good files (master) but not in the check files (worker).
     missing_files = {key: good_files[key] for key in good_files.keys() - check_files.keys()}
 
-    # Extra files are the ones present in check files (worker) but not in good files (master) and aren't extra valid.
-    extra_valid, extra = split_on_condition(check_files.keys() - good_files.keys(),
-                                            lambda x: cluster_items[check_files[x]['cluster_item_key']]['extra_valid'])
+    # Extra files are the ones present in check files (worker) but not in good files (master). The underscore is used
+    # to not change the function, as previously it returned an iterator for the 'extra_valid' files as well, but these
+    # are no longer in use.
+    _, extra = split_on_condition(check_files.keys() - good_files.keys(),
+                                  lambda x: cluster_items[check_files[x]['cluster_item_key']]['extra_valid'])
     extra_files = {key: check_files[key] for key in extra}
-    extra_valid_files = {key: check_files[key] for key in extra_valid}
-    if extra_valid_files:
+    _files = {key: check_files[key] for key in _}
+    # This condition should never take place. The 'PATH' string is a placeholder to indicate the type of variable that
+    # we should place.
+    if _files:
```

Comment:
The objective of this issue is to deactivate the extra-valid files mechanism, for that the ideal would be to isolate this functionality and simply not make use of it. 

The changes in the comments are fine, changing extra-valid by a placeholder is correct but only in the comments that can not be isolated.